components/shared/ConnectWalletButton.vue

```diff
@@ -21,17 +20,7 @@ export default class ConnectWalletButton extends Vue {
   private modal: { close: () => void; isActive?: boolean } | null = null
 
   public toggleWalletConnectModal(): void {
-    if (this.modal?.isActive) {
-      this.modal.close()
-      this.modal = null
-      return
-    }
-    this.modal = this.$buefy.modal.open({
-      parent: this,
-      ...ConnectWalletModalConfig,
-    })
-
-    this.$emit('closeBurgerMenu')
```

Comment:
pls do not delete this stuff. The changes will break the functionality to open the modal on mobile.

Reply:
oh, can we add mobile check then? we dont need this on web, its what was breaking this issue

Reply:
Is there any better way to handle this case? The function name is `toggleWalletConnectModal` which is not related to mobile check I guess : )